apps/src/templates/sectionProgress/VirtualizedSummaryView.jsx

```diff
@@ -14,6 +15,10 @@ export const progressStyles = {
     margin: '9px 16px',
     fontFamily: '"Gotham 5r", sans-serif',
   },
+  lessonOfInterest: {
+    fontSize: 20,
+    textShadow: '1px 1px 0px' + color.teal,
```

Comment:
I like the idea of letting the user see where they are, but this style isn't consistent with other places on our site. 

Can we remove the textShadow for now, and then in a later PR look at making the whole column "highlighted" or maybe light gray? I think pulling in some design feedback from Tanya/Mark would be good for this one.